samples/scala/src/main/scala/Main.scala

```diff
@@ -0,0 +1,20 @@
+import java.util.Calendar
+
+object Main extends App {
+  println("Hello, World!")
```

Comment:
Sorry to nit pick but could we remove this println

Reply:
My bad :) It's gone now